pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -680,6 +680,44 @@ type DatacenterSpecKubevirt struct {
 	// DNSConfig represents the DNS parameters of a pod. Parameters specified here will be merged to the generated DNS
 	// configuration based on DNSPolicy.
 	DNSConfig *corev1.PodDNSConfig `json:"dnsConfig,omitempty"`
+
+	// Images represents standard VM Image source.
+	Images *Images `json:"images,omitempty"`
+}
+
+var (
+	SupportedKubeVirtOS = map[providerconfig.OperatingSystem]*struct{}{
+		providerconfig.OperatingSystemCentOS:     nil,
+		providerconfig.OperatingSystemUbuntu:     nil,
+		providerconfig.OperatingSystemRHEL:       nil,
+		providerconfig.OperatingSystemFlatcar:    nil,
+		providerconfig.OperatingSystemRockyLinux: nil,
+	}
+)
+
+// Images represents standard VM Image source.
+type Images struct {
+	// HTTP source for VM disk Image.
+	HTTP *HTTP `json:"http,omitempty"`
+}
+
+// OSVersion defines a map of OS version and the URL to download the image.
+type OSVersion map[string]string
+
+// HTTP represents list of standard VM images with http-source.
+type HTTP struct {
+	// OperatingSystems represents list of supported operating-systems.
+	OperatingSystems map[providerconfig.OperatingSystem]OSVersion `json:"operatingSystems"`
+	// Optional: ImageCloning represents options for kubevirt disk-image cloning.
+	ImageCloning *ImageCloning `json:"imageCloning,omitempty"`
+}
+
+// ImageCloning represents options for kubevirt disk-image cloning.
+type ImageCloning struct {
+	// PreAllocatedDataVolume represent to use cached-images or not for cloning.
```

Comment:
s/PreAllocatedDataVolume/Enable
I would propose something like:
// Enable allows to enable/disable the option to clone disk images (defaults to false)